cmd/storj-sim/network.go

```diff
@@ -269,6 +269,11 @@ func newNetwork(flags *Flags) (*Processes, error) {
 			)
 		}
 
+		if flags.Redis != "" {
+			process.Arguments["setup"] = append(process.Arguments["setup"],
+				"--server.revocation-dburl", flags.Redis)
```

Comment:
If `flags.Redis` is supposed to be used "for all databases that support it" we probably need to use different dbs (`?db=0`, `?db=1`, etc.) for the different databases, otherwise I think all keys will share a namespace which could be problematic.

Reply:
Yeah definitely, I am opening another PR to be able to designate different namespaces for difference processes. So we can have revocationdb=db0, liveaccounting=db1...

Reply:
I can try to get that in before this pr if that's more strategic

Reply:
Maybe just leave a todo for now then?

Reply:
sounds good!